Demo/Demo/SwiftUIComponents/FeatureSelectionView.swift

```diff
@@ -38,10 +38,10 @@ struct FeatureSelectionView: View {
                         Text("Card Vaulting")
                     }
                     NavigationLink {
-                        PayPalWebView()
-                            .navigationTitle("PayPalWeb Payment")
+                        PayPalWebDemoView()
```

Comment:
The previous name containing webview could be confusing - updated the entry point to the feature to be called "demo view" but open to other names if folks prefer something else

Reply:
✌️cents: On Android we do lean toward `<FEATURE_NAME>View` because the Demo app is basically a contrived sample merchant integration for us to debug and QA the SDK. The fundamental idea is to create a Demoable "view" of each feature. The `FeatureName` prefix also scales nicely as a naming convention in other parts of the project.

Also, I believe the web team has a separate sample app with fictitious products to serve as a more complete onboarding example for merchants. Not super related, but worth mentioning.